upgrade_test.py

```diff
@@ -1438,6 +1443,38 @@ def _update_argus_upgraded_version(self, node, new_version):
         except Exception as exc:  # pylint: disable=broad-except
             self.log.exception("Failed to save upgraded Scylla version in Argus", exc_info=exc)
 
+    def validate_limited_voters_feature_enabled(self):
+        InfoEvent(message="Check that limited voters feature enabled after upgrade for all nodes")
```

Comment:
You are printing message before calling this function and right at the beginning of it. I would call print the message only once.

Reply:
fixed